### PR TITLE
feat(proposal): create flow

### PR DESCRIPTION
**Pull request by DarkNebula0**

### CODE REVIEW

File: pkg/app/src/components/Forms/Proposal/form.tsx [hunk 1]
```diff
@@ -85,9 +133,20 @@ export function Form({ currentStep, setStep }: ComponentProps) {
 				</Button>
 
 				<Button size="large" variant="contained" onClick={handleNext} disabled={checkNextButtonState()}>
-					{currentStep === 2 ? 'Save Proposal' : 'Next step'}
+					{currentStep === 2 ? 'Create Proposal' : 'Next step'}
```
Comment (developer-sidani) [inline pkg/app/src/components/Forms/Proposal/form.tsx:136]: @DarkNebula0 don't you think it's better to create it as a variable at the top of the function and use the useMemo hook for that, or not necessary?

File: pkg/app/src/components/Forms/Proposal/modules/majority.tsx [hunk 2]
```diff
@@ -39,9 +43,9 @@ export function Majority({ selected, setSelected, deposit, setDeposit }: Compone
 				value={1}
 				selectedValue={selected}
 				onChange={setSelected}
-				disabled={true}
+				disabled={!enabledFeature.CREATE_PROPOSAL_RELATIVE_MAJORITY}
```
Comment (developer-sidani) [inline pkg/app/src/components/Forms/Proposal/modules/majority.tsx:46]: better with a variable with the useMemo callback, what do you think?

File: pkg/app/src/components/Layouts/default/modules/header.tsx [hunk 3]
```diff
@@ -20,15 +20,15 @@ interface ComponentProps {
 const urls = [
```
Comment (developer-sidani) [inline pkg/app/src/components/Layouts/default/modules/header.tsx:20]: useMemo callback

File: pkg/app/src/hooks/tx/useCreateOrgTransaction.ts [hunk 4]
```diff
@@ -1,11 +1,12 @@
...
-import { useTmpOrganisation } from 'hooks/useTmpOrganisation'
 import { useCurrentAccountAddress } from 'hooks/useCurrentAccountAddress'
+import { useTmpOrganisation } from 'hooks/useTmpOrganisation'
+import { useNetworkContext } from 'provider/network/modules/context'
 import { fromUnit } from 'src/utils/token'
 import { encode as utf8Encode } from 'utf8'
+import * as Yup from 'yup'
 
 const validation = Yup.object().shape({
 	controller_id: Yup.string().required(),
```
Comment (developer-sidani) [inline pkg/app/src/hooks/tx/useCreateOrgTransaction.ts:12]: Isn't it better to have a single file for validation schema?

File: pkg/app/src/components/Forms/Proposal/form.tsx (hunk 1 above)
Comment (DarkNebula0) [reply pkg/app/src/components/Forms/Proposal/form.tsx:136]: useMemo is not necessary for this but it should actually come through the translation file. I will change it.

**Pull request merged**